wiki/cuda2alpaka.md

```diff
@@ -0,0 +1,55 @@
+## A first practical example
+
+### Try it yourself
+
+If you want to try to build and run the example you can find the code at: https://github.com/cms-patatrack/pixeltrack-standalone.git
+
+The first step is to change the version of ```gcc``` to 8.3.1.
+
+```bash
+source scl_source enable devtoolset-8
+```
+
+Then, you need to download the repository.
+
+```bash
+# Clone the repository
+git clone git@github.com:cms-patatrack/pixeltrack-standalone.git
+
+# Change the current working directory
+cd pixeltrack-standalone/
+```
+
+The code can be compiled for different backends. In order to compile it for CUDA or Alpaka, the commands are:
+
+```bash
+# Build application with N-fold concurrency using cuda-11.3
+make -j N cuda
```

Comment:
```suggestion
make -j $(nproc) cuda
```

If you use `$(nproc)` instead of `N`, the instructions remain general, and can be copy-pated directly into a terminal.